pom.xml

```diff
@@ -557,6 +558,11 @@
         <artifactId>jersey-container-servlet-core</artifactId>
         <version>${glassfish.version}</version>
       </dependency>
+      <dependency>
```

Comment:
This is added to do away with warnings like below:
`The POM for org.glassfish:javax.el:jar:3.0.1-b06-SNAPSHOT is missing, no dependency information available`

Reply:
ok but does this have to be the default compile scope ?